### ISSUE TITLE
pretrained models

### ISSUE THREAD

**Issue by seekingdeep** [dingmyu/HR-NAS#2]
@dingmyu hi,
were the pretrained models uploaded such as for the imagenet and cityscapes.
can it run inferencing using cpu-only
can hr-nas detect small objects

**Comment by dingmyu** [comment]
Hi,

- The pre-trained model is currently unavailable. Please stay tuned.
- The searched model can run in CPU-only mode, but its efficiency (which cannot run in real-time) will be much slower than GPU.
- Sample images can be found in the paper. We have not evaluated the performance of detecting small objects because HR-NAS aims to search for efficient models.

Thanks,
Mingyu

**Comment by seekingdeep** [comment]
last question here:
- have you considered pruning the final efficient models, inorder to reduce the flops and parameter?
i can send you some links to interesting spare pruning methods if you want.
- have you considered exporting to onnx so it can run on arm devices such as raspberry pi?

**Comment by dingmyu** [comment]
> last question here:
> 
> * have you considered pruning the final efficient models, inorder to reduce the flops and parameter?
>   i can send you some links to interesting spare pruning methods if you want.
> * have you considered exporting to onnx so it can run on arm devices such as raspberry pi?

Hi,

- The search algorithm of HR-NAS is based on pruning. And we tried to manually reduce some channels of the searched model, the accuracy drops a little.
-  We have not tried the model on chips or arms and do not have the plan of exporting to onnx, thanks. PR is welcomed.

Thanks.

**Comment by seekingdeep** [comment]
thanks
